swagger.yaml

```diff
@@ -424,6 +424,39 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/on-call/users/{username}":
```

Comment:
Do we really need an enpoint and entire lambda for this? Isn't this something that could be done in the existing on call fetch/ listing?